index.js

```diff
@@ -1,17 +1,46 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  get(pos) {
 
-  max() {}
+    if (pos >= this.length) throw new Error('OutOfBounds');
+    return this.items[pos];
+  }
 
-  min() {}
+  max() {
 
-  sum() {}
+    if (this.length === 0) throw new Error('EmptySortedList');
+    return this.items[this.length - 1];
 
-  avg() {}
+  }
+
+  min() {
+    if (this.length === 0) throw new Error('EmptySortedList');
+    return this.items[0];
+  }
+
+  sum() {
+
+    if (this.length === 0) return 0;
+    return this.items.reduce((acc, curVal) => acc + curVal);
```

Comment:
one quick thing, you can try to loop over an empty array. 
If you reduce like this:
```js
return this.items.reduce((acc,curVal) => acc + curVal, 0)
```
Even if the array is empty, the reduce will default to its "initial value", in this case, the 0